packages/react-components/react-drawer/tsconfig.lib.json

```diff
@@ -16,7 +16,9 @@
     "**/*.test.ts",
     "**/*.test.tsx",
     "**/*.stories.ts",
-    "**/*.stories.tsx"
+    "**/*.stories.tsx",
+    "**/*.cy.ts",
```

Comment:
wondering why is this part of this PR. Bad branch update/ bug in generator ?

this should be removed anyways